### PR TITLE
helloWorld ide Plugin 

### PR DESCRIPTION
**Pull request by i-walker**
Aims to solve #210

### CODE REVIEW

File: idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/helloworld/DummyPlugin.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ * The following section exemplifies a Hello World Ide Plugin
+ *
+ * The Hello World plugin registers a LineMarker on every `helloWorld` function.
+ *
+ * ```kotlin
+ * val IdeMetaPlugin.helloWorld: Plugin
+ *    get() = "Hello World" {
+ *      meta(
+ *        addLineMarkerProvider(
+ *          icon = ArrowIcons.PURE,
```
Comment (franciscodr) [inline idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/helloworld/DummyPlugin.kt]: I guess we will need a specific icon for the hello world plugin

**Comment by i-walker** [comment]
@israelperezglez may you link here the Icon I should add for the HelloWorld plugin

File: idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/helloworld/DummyPlugin.kt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ *            }
+ *          }
+ *        )
+ *      )
+ *    }
+ * ```
+ *
+ * For every function with the name `helloWorld` our ide plugin will register a lineMarker with our costume icon and whenever
+ * the user hovers over the Icon it will display the message.
+ * // TODO: Add Animation or example picture
```
Comment (JorgeCastilloPrz) [inline idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/helloworld/DummyPlugin.kt]: Is this gonna be done in this PR?

**Comment by franciscodr** [comment]
@i-walker I have already talked to Isra and he's gonna add that icon in another issue that he's working on to create new Arrow Meta icons for the type proofs. I'll let you know when it's ready

**Comment by i-walker** [comment]
@franciscodr @israelperezglez Thanks to both of you :) This PR is good to go once we have the Icon. @raulraja does the description work for you ?

File: idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/helloworld/DummyPlugin.kt (hunk 2 above)
Comment (i-walker) [reply idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/helloworld/DummyPlugin.kt]: Probably, not. We can add it later

**Pull request merged**